src/main/java/com/playonlinux/framework/SetupWizard.java

```diff
@@ -103,6 +109,47 @@ public void execute(Message message) {
                 }
         );
     }
+
+    /**
+     * Show the content of a licence file
+     * @param textToShow a message above the licence
+     * @param licenceFile the licence file to display (with 'from java.io import File')
+     * @throws ScriptFailureException
+     * @throws CancelException
+     */
+    public void licence(String textToShow, File licenceFile) throws ScriptFailureException, CancelException {
+        try {
+            final FileInputStream content = new FileInputStream(licenceFile);
+            final StringWriter writer = new StringWriter();
+            IOUtils.copy(content, writer, "UTF-8");
+            content.close();
+            showLicense(textToShow, writer.toString());
+        } catch (IOException e) {
+            throw new ScriptFailureException("Cannot acces the licence file", e);
+        }
+    }
+
+    /**
+     * Show a custom licence message
+     * @param textToShow a message above the licence
+     * @param licenceText the licence text to show
+     * @throws CancelException
+     */
+    public void licence(String textToShow, String licenceText) throws CancelException {
```

Comment:
and here the (string,string) version, the python script no need modification to use it (more simple to use it that the other)